pkg/operator/controller/garden/garden/reconciler_delete.go

```diff
@@ -67,29 +67,36 @@ func (r *Reconciler) delete(
 		})
 		destroyVirtualGardenGardenerAccess = g.Add(flow.Task{
 			Name: "Destroying Gardener virtual garden access resources",
-			Fn:   c.virtualGardenGardenerAccess.Destroy,
-		})
-		destroyVirtualGardenGardenerResourceManager = g.Add(flow.Task{
-			Name:         "Destroying gardener-resource-manager for virtual garden",
-			Fn:           component.OpDestroyAndWait(c.virtualGardenGardenerResourceManager).Destroy,
-			Dependencies: flow.NewTaskIDs(destroyVirtualGardenGardenerAccess),
+			Fn:   component.OpDestroyAndWait(c.virtualGardenGardenerAccess).Destroy,
```

Comment:
Could you please enlighten me what this change does? As far as I can tell, `virtualGardenGardenerAccess` has no `Wait` or `WaitCleanup` functions.

Reply:
Good point, I did not add these functions to `gardeneraccess` yet. Here they are 😄